tests/utils/conda_package_helper.py

```diff
@@ -41,58 +40,36 @@ class CondaPackageHelper:
 
     def __init__(self, container: TrackedContainer):
-        self.running_container: Container = CondaPackageHelper.start_container(
-            container
-        )
-        self.requested: dict[str, set[str]] | None = None
-        self.installed: dict[str, set[str]] | None = None
-        self.available: dict[str, set[str]] | None = None
-        self.comparison: list[dict[str, str]] = []
-
-    @staticmethod
-    def start_container(container: TrackedContainer) -> Container:
-        """Start the TrackedContainer and return an instance of a running container"""
+        self.container = container
         LOGGER.info(f"Starting container {container.image_name} ...")
-        return container.run_detached(
+        container.run_detached(
             tty=True,
             command=["bash", "-c", "sleep infinity"],
         )
 
-    @staticmethod
-    def _conda_export_command(from_history: bool) -> list[str]:
-        """Return the mamba export command with or without history"""
-        cmd = ["mamba", "env", "export", "--no-build", "--json"]
-        if from_history:
-            cmd.append("--from-history")
-        return cmd
+        self.requested: dict[str, set[str]] | None = None
+        self.installed: dict[str, set[str]] | None = None
+        self.available: dict[str, set[str]] | None = None
+        self.comparison: list[dict[str, str]] = []
 
     def installed_packages(self) -> dict[str, set[str]]:
         """Return the installed packages"""
         if self.installed is None:
             LOGGER.info("Grabbing the list of installed packages ...")
-            self.installed = CondaPackageHelper._parse_package_versions(
-                self._execute_command(
-                    CondaPackageHelper._conda_export_command(from_history=False)
-                )
-            )
+            env_export = self.container.exec_cmd("mamba env export --no-build --json")
+            self.installed = CondaPackageHelper._parse_package_versions(env_export)
         return self.installed
 
     def requested_packages(self) -> dict[str, set[str]]:
         """Return the requested package (i.e. `mamba install <package>`)"""
         if self.requested is None:
             LOGGER.info("Grabbing the list of manually requested packages ...")
-            self.requested = CondaPackageHelper._parse_package_versions(
-                self._execute_command(
-                    CondaPackageHelper._conda_export_command(from_history=True)
-                )
+            env_export = self.container.exec_cmd(
+                "mamba env export --no-build --json --from-history"
             )
+            self.requested = CondaPackageHelper._parse_package_versions(env_export)
         return self.requested
 
-    def _execute_command(self, command: list[str]) -> str:
-        """Execute a command on a running container"""
-        exec_result = self.running_container.exec_run(command, stderr=False)
```

Comment:
Removed stderr here, because mamba was updated recently and now the output is clean (as it was before)